plugins/webapi/info/plugin.go

```diff
@@ -100,11 +100,15 @@ func getInfo(c echo.Context) error {
 	sort.Strings(disabledPlugins)
 
 	// get TangleTime
-	lcm := deps.Tangle.TimeManager.LastConfirmedMessage()
+	tm := deps.Tangle.TimeManager
+	lcm := tm.LastConfirmedMessage()
 	tangleTime := jsonmodels.TangleTime{
-		Synced:    deps.Tangle.TimeManager.Synced(),
-		Time:      lcm.Time.UnixNano(),
+		Synced:    deps.Tangle.TimeManager.Bootstrapped(),
```

Comment:
Update the property name to 'Boostrapped' to have consistent naming. I think it would be nice to have another field 'synced' that is based on the difference  RCTT and CTT for API stuff and monitoring metrics.

Reply:
Done